packages/terra-button-group/src/ButtonGroupButton.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { PropTypes } from 'react';
```

Comment:
I think this will be easier to maintain if we treat this component as an h.o.c. that wraps the `<Button />` component. This way we don't have to maintain duplicate "button" components / styles.